AsyncDisplayKit/TextKit/ASTextKitRenderer.mm

```diff
@@ -203,42 +238,50 @@ - (void)drawInContext:(CGContextRef)context bounds:(CGRect)bounds;
     [self _calculateSize];
   }
 
+  bounds = CGRectIntersection(bounds, { .size = _constrainedSize });
```

Comment:
This wasn't needed before, because the slow-path drawing actually doesn't use `bounds`. It uses `textContainer.size` which got updated when the constrained size changed to account for the text inset. Since the fast-path does use `bounds`, this is necessary. We could absolutely refactor this, to have the constrained (inset) bounds passed into this method but time is of the essence.